src/widgets/trilium/component.jsx

```diff
@@ -0,0 +1,48 @@
+import Block from "components/services/widget/block";
+import Container from "components/services/widget/container";
+import { useTranslation } from "next-i18next";
+import { FiFileText, FiPaperclip } from "react-icons/fi";
+import { RiStackLine } from "react-icons/ri";
+
+import useWidgetAPI from "utils/proxy/use-widget-api";
+
+export default function Component({ service }) {
+  const { t } = useTranslation();
+  const { widget } = service;
+
+  // Fetch metrics from Trilium in JSON format
+  const { data: metricsData, error: metricsError } = useWidgetAPI(widget, "metrics", {
+    refreshInterval: 60000, // refresh every minute
+  });
+
+  if (metricsError) {
+    return <Container service={service} error={metricsError} />;
+  }
+
+  if (!metricsData) {
+    return (
+      <Container service={service}>
+        <Block label="trilium.version" />
+        <Block label="trilium.notesCount" />
+        <Block label="trilium.attachmentsCount" />
+      </Container>
+    );
+  }
+
+  // Extract values from the clean JSON structure
+  const version = metricsData?.version?.app;
+  const notesCount = metricsData?.database?.activeNotes || 0;
+  const attachmentsCount = metricsData?.database?.activeAttachments || 0;
```

Comment:
```suggestion
  const version = metricsData.version?.app;
  const notesCount = metricsData.database?.activeNotes || 0;
  const attachmentsCount = metricsData.database?.activeAttachments || 0;
```

`metricsData` definitely exists at this point